packages/edit-site/package.json

```diff
@@ -54,6 +54,7 @@
 		"@wordpress/style-engine": "file:../style-engine",
 		"@wordpress/url": "file:../url",
 		"@wordpress/viewport": "file:../viewport",
+		"@wordpress/widgets": "file:../widgets",
```

Comment:
How fine are we with this dependency? Are there any risks of side effects or anything?

Also by adding this dependency to block-library like suggested above, we'll be also loading this script in the post editor, do we want that?

Reply:
The package has no side effects.

> Also by adding this dependency to block-library like suggested above, we'll be also loading this script in the post editor, do we want that?

This has been decided yet. I think duplicating `transformWidgetToBlock` is better than creating an early abstraction.

Reply:
Added the `@wordpress/widgets` package directly to the `edit-post`. But let's decide if we want to move it into the block library package.